server/views/serviceProviderReferrals/partials/layoutAllComponents.njk

```diff
@@ -0,0 +1,47 @@
+{% extends "../templates/serviceProviderSessionsTemplate.njk" %}
```

Comment:
I don't think I understand why we have two files to create this layout. Would you mind explaining the difference between `partials/layoutAllComponents` and `templates/serviceProviderSessionsTemplate.njk`? Could we not just put all of this into the template file?  I thought that a partial is a component that you drop into a larger page, not something that defines the layout of the entire page.